app/components/SimplePagination.tsx

```diff
@@ -0,0 +1,38 @@
+'use client';
+
+import { cx } from 'class-variance-authority';
+
+import { Button, Icon } from '@swapr/ui';
+
+interface SimplePaginationProps {
+  isFinalPage: boolean;
+  page?: number;
+  setPage?: (page: number) => void;
+}
+
+export const SimplePagination = ({
+  isFinalPage,
+  page,
+  setPage,
+}: SimplePaginationProps) =>
+  page && (page !== 1 || !isFinalPage) && setPage ? (
```

Comment:
instead of ternary can we have an early return? 

I think it's cleaner to read

`!(page && (page !== 1 || !isFinalPage) && setPage) return null`